src/components/Forms/EpicNominationForm/NominateRatForm.js

```diff
@@ -0,0 +1,94 @@
+/* eslint-disable react/jsx-no-bind */
+import { useEffect, useState } from 'react'
+
+import InputFieldset from '~/components/Fieldsets/InputFieldset'
+import RatTagsInput from '~/components/RatTagsInput'
+import getRatTag from '~/helpers/getRatTag'
+import { isValidUuidV4 } from '~/helpers/uuidValidator'
+import useCreateEpicApi from '~/hooks/frapi/useCreateEpicApi'
+import useForm from '~/hooks/useForm'
+
+import styles from './NominateRatForm.module.scss'
+
+
+export default function NominateRatForm ({ onSuccess, onError }) {
+  const [userId, setUserId] = useState(null)
+
+  const [{
+    isLoading: isCreateLoading,
+    isError: isCreateError,
+    epicId: createdEpicId,
+  }, createEpic] = useCreateEpicApi()
+
+  const onSubmit = (props) => {
+    createEpic({
+      notes: props.attributes.notes,
+      nomineeIds: [userId],
+    })
+  }
+
+  const onChangeRat = (rats) => {
+    if (rats.length !== 1 || !isValidUuidV4(rats[0].relationships.user.data.id)) {
+      setUserId(null)
+      return
+    }
+
+    setUserId(rats[0].relationships.user.data.id)
+  }
+  const {
+    Form,
+    submitting,
+    canSubmit,
+  } = useForm({
+    data: {
+      attributes:
+        {
+          rat: '',
+          notes: undefined,
+        },
+    },
+    onSubmit,
+  })
+
+  useEffect(() => {
+    if (typeof createdEpicId === 'string') {
+      onSuccess()
+    }
+  }, [createdEpicId, onSuccess])
+
+  useEffect(() => {
+    if (isCreateError) {
+      onError()
+    }
+  }, [isCreateError, onError])
+
+  return (
+    <Form>
+
+      <fieldset>
+        <label htmlFor="rat">{"What is the rat's CMDR name?"}</label>
+
+        <RatTagsInput
+          data-single
+          required
+          aria-label="Commander name"
+          disabled={submitting}
+          id="rat"
+          name="attributes.rat"
+          valueProp={getRatTag}
+          onChange={onChangeRat} />
+      </fieldset>
+
+      <InputFieldset required id="notes" label="What did the rat do that makes him so epic?" name="attributes.notes" />
```

Comment:
```suggestion
      <InputFieldset required id="notes" label="What did the rat do that makes them so epic?" name="attributes.notes" />
```

Please prefer gender neutral pronouns 😄